O21.Resources/Graphics.fs

```diff
@@ -5,36 +5,41 @@ open System.IO
 
 open O21.NE
 
-let Load(path: string): Dib[] =
+let Load (path: string) : Dib[] =
     use stream = new FileStream(path, FileMode.Open, FileAccess.Read)
     let neFile = NeFile.ReadFrom stream
     let resources = neFile.ReadResourceTable()
+
     let bitmapResourceType =
-        resources
-        |> Seq.filter (fun x -> x.TypeId = 32770us)
-        |> Seq.exactlyOne
+        resources |> Seq.filter (fun x -> x.TypeId = 32770us) |> Seq.exactlyOne
+
     bitmapResourceType.Resources
     |> Seq.map neFile.ReadResourceContent
     |> Seq.map Dib
     |> Seq.toArray
 
-let Export (outDir: string) (images: Dib seq): unit =
+let Export (outDir: string) (images: Dib seq) : unit =
     Directory.CreateDirectory outDir |> ignore
 
     let mutable i = 0
+
     for dib in images do
         let pixelDataOffset =
-            14 + // BMP header
-            40 + // DIB header
-            dib.Palette.Length * 4 // palette
-        let fileContent =
-            [|
-                yield! "BM"B
-                yield! BitConverter.GetBytes(dib.Raw.Length + 14)
-                yield! BitConverter.GetBytes 0us
-                yield! BitConverter.GetBytes 0us
-                yield! BitConverter.GetBytes pixelDataOffset
-                yield! dib.Raw
-            |]
+            14
+            + // BMP header
+            40
+            + dib // DIB header
+              .Palette.Length
+              * 4 // palette
```

Comment:
Ok, this is really weird. Let's move the comments above the operator, I guess? And definitely I'll create an issue in Fantomas about this.